<!-- -->
<launch>

	<include file="$(find model_publisher)/launch/vehicle_model.launch">
	<arg name="topic_name" default="sim_vehicle_model"/>
	<arg name="base_frame" default="sim_base_link"/>
	</include>
	
	<arg name="use_pose" default="InitialPos"/>
    <node pkg="waypoint_follower" type="odom_gen" name="odom_gen" output="screen">
    	<param name="use_pose" value="$(arg use_pose)"/>
    </node>

</launch>